### PR TITLE
[Test Automation] T1 Community Home Page and About Page Test & Initial Testing Resources

### PR DESCRIPTION
**Pull request by Julian88Tex**
# Critical Changes

# Changes
- created an automated test to check for home page and about us page contents

# Issues Closed

### CODE REVIEW

**Comment by Julian88Tex** [comment]
@skristem @rjanjanam this is ready for review :)

File: robot/Crisis-Management/tests/browser/t5_user_able_to_update_availibility_staff.robot [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+    Click Element                           //*/span[1]/label/span[@class="slds-radio_faux"]  #Not Available Option
+    Click Element                           //*//div/a[2]/span[@class="slds-m-left_x-small"]  #OK Button
+    SeleniumLibrary.Element Text Should Be  //*/li[1]/c-staff-availability-item/div/div/div[2]/div/div[1][@class="cmp-availability"]  Not Available  #1st Availibility Button
+
+Set 4th Availibility to Availible at Julian Account
+    Click Element                           //*/li[4]/c-staff-availability-item/div/div/div[2]/div/div[@data-available="Available"]  #4th Availibility Button
+    Click Element                           //*/li[4]/c-staff-availability-item/div/div[2]/div[1]/div/lightning-radio-group/fieldset/div/div/span[3]/label/span[@class="slds-radio_faux"]  #Assigned Option
+    Click Element                           //*/li[4]/c-staff-availability-item/div/div[2]/div[2]/div/a[2]/span[@class="slds-m-left_x-small"]  #OK Button
+    SeleniumLibrary.Element Text Should Be  //*/li[4]/c-staff-availability-item/div/div/div[1]/div[3]/span[@class="cmp-facility-name"]  Julian Account  #Account
```
Comment (sgovindankutty) [inline robot/Crisis-Management/tests/browser/t5_user_able_to_update_availibility_staff.robot]: Please move all the locators to locators.py or locators_47.py file. The locators mentioned here might not be reliable as the indices of these might keep changing with the change in application. We can go over in tomorrow’s call on this if you are interested. For Validations, it is good to practice to wrap it in a function and refer the function/keyword in the robot file.

Comment (Julian88Tex) [reply robot/Crisis-Management/tests/browser/t5_user_able_to_update_availibility_staff.robot]: @sgovindankutty yeah would love to discuss these best practices during the meeting :-)

File: tasks/check_rd2_enablement.py [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+            self.logger.info("Identified Enhanced Recurring Donations status: {}".format(self.return_values))
+            return
+
+        if settings.get("records"):
+            if settings["records"][0]["npsp__IsRecurringDonations2Enabled__c"]:
+                self.return_values = True
+                self.logger.info("Identified Enhanced Recurring Donations status: {}".format(self.return_values))
+                return
+
+        self.return_values = False
```
Comment (rjanjanam) [inline tasks/check_rd2_enablement.py]: Why do you need RD2 enabled check for this testcase ?

File: robot/Crisis-Management/tests/browser/t2_user_able_to_update_contact_and_credentials_home.robot [hunk 3]
```diff
@@ -0,0 +1,73 @@
+*** Settings ***
+
+Resource        robot/Crisis-Management/resources/NPSP/NPSP.robot
+Resource        robot/Crisis-Management/resources/CrisisManagement.robot
+Library         cumulusci.robotframework.PageObjects
+...             robot/Crisis-Management/resources/NPSP/ContactPageObject.py
+Suite Setup     Run keywords
+...             Open Test Browser
+# Suite Teardown  Delete Records and Close Browser
```
Comment (rjanjanam) [inline robot/Crisis-Management/tests/browser/t2_user_able_to_update_contact_and_credentials_home.robot]: Why is this commented out?

File: robot/Crisis-Management/tests/browser/t2_user_able_to_update_contact_and_credentials_home.robot [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+
+Set 4th Availibility to Availible at Julian Account
+    Click Element                           //*/li[4]/c-staff-availability-item/div/div/div[2]/div/div[@data-available="Available"]  #4th Availibility Button
+    Click Element                           //*/li[4]/c-staff-availability-item/div/div[2]/div[1]/div/lightning-radio-group/fieldset/div/div/span[3]/label/span[@class="slds-radio_faux"]  #Assigned Option
+    Click Element                           //*/li[4]/c-staff-availability-item/div/div[2]/div[2]/div/a[2]/span[@class="slds-m-left_x-small"]  #OK Button
+    SeleniumLibrary.Element Text Should Be  //*/li[4]/c-staff-availability-item/div/div/div[1]/div[3]/span[@class="cmp-facility-name"]  Julian Account  #Account
+
+
+*** Test Cases ***
+
```
Comment (rjanjanam) [inline robot/Crisis-Management/tests/browser/t2_user_able_to_update_contact_and_credentials_home.robot]: Please add some description for the objective of this testcase

File: robot/Crisis-Management/resources/ContactPageObject.py [hunk 5]
```diff
@@ -0,0 +1,106 @@
...
+            locator_login_link,
+            error="'Log in to community as user' option is not available in the list of actions"
+        )
+        self.selenium.click_element(locator_login_link)
+        self.selenium.wait_until_location_contains("/TractionThrive/s/",timeout=60, message="Community page did not load in 1 min")
+        landing_tab=cm_lex_locators["link-text"].format("Home")
+        status=self.selenium.get_webelement(landing_tab).get_attribute("class")
+        assert "active" in status, "Community landing tab is not Home"
+
+    def enable_community_login(self):
```
Comment (skristem) [inline robot/Crisis-Management/resources/ContactPageObject.py]: I think u can delete this as we now have api solution

**Comment by Julian88Tex** [comment]
@camgood-trac This is ready for review and/or merge :-)

**Review by ghost — APPROVED** [review]

**Pull request merged**